content/en/blog/_posts/2021-12-21-admission-controllers-for-container-drift/index.md

```diff
@@ -68,7 +68,19 @@ As you can see in the above event messages, the affected Pod is not evicted imme
 For our production clusters, we specify a lower time limit so as to avoid the impacted Pods serving traffic abidingly. The *kube-exec-controller* internally sets and tracks a timer for each Pod that matches the associated TTL. Once the timer is up, the controller evicts that Pod using K8s API. The eviction (rather than deletion) is to ensure service availability, since the cluster respects any configured [PodDisruptionBudget](/docs/concepts/workloads/pods/disruptions/) (PDB). Let's say if a user has defined *x* number of Pods as critical in their PDB, the eviction (as requested by *kube-exec-controller*) does not continue when the target workload has fewer than *x* Pods running.
 
 Here comes a sequence diagram of the entire workflow mentioned above: 
-{{< figure src="workflow-diagram.svg" alt="Workflow Diagram" class="diagram-medium" >}}
+{{<mermaid>}}
+sequenceDiagram
+actor me as <br>Developer (service-owner)
```

Comment:
Is there a reason for hyphenating service-owner? If it refers to an object name, then okay. In the original diagram, it just looks like it is referring to the person who owns the service.